src/index.html

```diff
@@ -8,10 +8,206 @@
+            name="age" 
+            min="1" 
+            max="100" 
+            value="12"
+            autocomplete="off" 
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input 
+            type="date" 
+            name="date" 
+            required
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="terms" 
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="group">
+
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com"
+            autocomplete="off" 
+            required
+          >
+        </label>
+
+        <label>
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            autocomplete="off" 
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="group">
+
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="no"
+            >
+            No
+          </label>
+        </div>  
+        
+        <label class="form-field">
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="color" 
+            required
+          >
+        </label>
+        
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="bed" 
+            min="09:00" 
+            max="18:00"
+            required
+          >
+        </label>
+        
+        <label class="form-field" >
+          What are your favorite brand of cars?
+
+          <select 
+            name="brand" 
+            required 
+            multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+
+        </label>
+
+        <label>
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="rate"
+            min="5"
+            max="50"
+            value="25"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="group">
+
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea 
+            name="Comments" 
+         ></textarea>
```

Comment:
Only when tag has more than 2 atrributes we use multilines
```suggestion
          <textarea name="Comments"></textarea>
```